web/app.js

```diff
@@ -164,6 +165,9 @@ let PDFViewerApplication = {
     });
 
     this.initialized = true;
+
+    // update url on scroll
```

Comment:
You can remove this comment and add it to the `setupHashUpdating` function below so it's clear what the objective of that function is.